otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -61,7 +62,9 @@
 from otx.algorithms.detection.adapters.mmdet.datasets import ImageTilingDataset
 from otx.algorithms.detection.adapters.mmdet.hooks.det_class_probability_map_hook import (
     DetClassProbabilityMapHook,
+    MaskRCNNRecordingForwardHook,
 )
+from otx.algorithms.detection.adapters.mmdet.models.detectors import CustomMaskRCNN
```

Comment:
```suggestion
```
I think it isn't used

Reply:
It was us nut I replaced it with `isinstance(feature_model, TwoStageDetector)`. Thanks!